outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
```

Comment:
I don't really see how modules would replace packages - in my mind the primary function of a package is to get other people's code, and modules don't really do that at all? IDK.